TheMovieDB/Movie.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  Movie.swift
+//  TheMovieDB
+//
+//  Created by Juliana Lima on 29/10/16.
+//  Copyright © 2016 Juliana Lacerda. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+import SwiftyJSON
+
+class Movie {
+    
+    var id: Int
+    var title: String
+    var overview: String?
+    var posterPath: String?
+    var backdropPath: String?
+    var releaseDate: Date?
+    var genres: [String]?
+    
+    var releaseDateString: String {
+        get {
+            let dayTimeFormatter = DateFormatter()
+            
+            dayTimeFormatter.dateFormat = "dd/MM/yyyy"
```

Comment:
The model class is not the best place to handle formatting. If i18n becomes a feature you would need to change  the model classes. They should be agnostic about this.